package.json

```diff
@@ -36,10 +36,10 @@
     "test:unit": "jest",
     "test:unit:watch": "pnpm test:unit --collectCoverage=false --watch",
     "test:playwright": "./bin/playwright.sh",
-    "test:playwright:local": "playwright test -c test/playwright",
+    "test:playwright:local": "DISABLE_SENTRY=true playwright test -c test/playwright",
     "test:playwright:debug": "PWDEBUG=1 pnpm test:playwright:local",
     "test:playwright:recreate-tapes": "rimraf test/tapes && pnpm test:playwright:update-tapes",
-    "test:playwright:update-tapes": "UPDATE_TAPES=true pnpm test:playwright",
+    "test:playwright:update-tapes": "UPDATE_TAPES=true DISABLE_SENTRY=true pnpm test:playwright",
```

Comment:
Is `DISABLE_SENTRY` option necessary in this one? It calls to `test:playwright` which uses the docker-compose where you also added the option.

Reply:
Oh right, thanks!